sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -109,6 +109,10 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
     }
     renderer.onStart()
     pluginRegistry.onStart()
+    if (!mapboxMap.isStyleLoadInited) {
```

Comment:
What will happen if user will load his style in `Activity.onStart()`? This will actually mean that we will already start loading `Style.MAPBOX_STREETS` from `mapView.onStart()`. Will everything work OK then? I guess desired behaviour is to result in user style loaded and not default one.

Reply:
User's style will be loaded instead of the default one.

Reply:
Without any glitches, correct?

Reply:
Yes, no glitches